packages/@aws-cdk/aws-dynamodb-global/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@aws-cdk/aws-dynamodb-global",
```

Comment:
Why do we need a new package as opposed to `@aws-cdk/aws-dynamodb`?

Reply:
Made more sense to me to have a new package since it has different requirements than the regular aws-dynamodb table - i.e. required `tableName` and `streamSpecification`.  As well as its a higher level object than the dynamodb package.

Once CFN officially supports global-dynamodb tables, it would probably be easier to deprecate this package than remove it from an integrated portion of the aws-dynamodb package

Reply:
I think it's fine to have this as a separate module. Maybe we can add a comment in the dynamodb README that mentions it